src/globals/defines.php

```diff
@@ -41,8 +41,9 @@
 ];
 
 // spc lock type
-const SPC_LOCK_SOURCE = 1;      // lock source
-const SPC_LOCK_PRE_BUILT = 2;   // lock pre-built
+const SPC_DOWN_SOURCE = 1;      // lock source
+const SPC_DOWN_PRE_BUILT = 2;   // lock pre-built
+const SPC_DOWN_PACKAGE = 3; // lock as package
```

Comment:
I think we should spell these out. I wasn't sure what DOWN referred to immediately.